kfs-core/src/main/resources/edu/arizona/kfs/fp/businessobject/datadictionary/PaymentMethod.xml

```diff
@@ -0,0 +1,219 @@
+<?xml version="1.0" encoding="UTF-8"?><beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:p="http://www.springframework.org/schema/p" xsi:schemaLocation="http://www.springframework.org/schema/beans         http://www.springframework.org/schema/beans/spring-beans-2.0.xsd">
+
+  <bean id="PaymentMethod" parent="PaymentMethod-parentBean"/>
+
+  <bean id="PaymentMethod-parentBean" abstract="true" parent="BusinessObjectEntry">
+    <property name="businessObjectClass" value="edu.arizona.kfs.fp.businessobject.PaymentMethod"/>
+    <property name="inquiryDefinition" ref="PaymentMethod-inquiryDefinition" />
+    <property name="lookupDefinition" ref="PaymentMethod-lookupDefinition" />
+    <property name="titleAttribute" value="paymentMethodCode"/>
+    <property name="objectLabel" value="Payment Method"/>
+    <property name="attributes">
+      <list>
+        <ref bean="PaymentMethod-paymentMethodCode"/>
+        <ref bean="PaymentMethod-paymentMethodName"/>
+        <ref bean="PaymentMethod-bankCode"/>
+        <ref bean="PaymentMethod-assessedFees"/>
+        <ref bean="PaymentMethod-processedUsingPdp"/>
+        <ref bean="PaymentMethod-interdepartmentalPayment"/>
+        <ref bean="PaymentMethod-offsetUsingClearingAccount"/>
+        <ref bean="PaymentMethod-displayOnDisbursementVoucherDocument"/>
+        <ref bean="PaymentMethod-displayOnVendorDocument"/>
+        <ref bean="PaymentMethod-active"/>
+      </list>
+    </property>
+  </bean>
+
+<!-- Attribute Definitions -->
+
+  <bean id="PaymentMethodAttribute-SelectControl" parent="PaymentMethod-paymentMethodCode" p:control-ref="PaymentMethodSelectControl" />
```

Comment:
I don't see where this bean is being referenced. Maybe not needed for step 1 of this mod.